google/ggrc-core
# GGRC-5784 Add base_id column to acl table

## ⚠️This PR does not have running tests! The tests get broken by this PR so reviewer should manually check If the developer missed any important issues! 

# Dependencies

This PR is `on hold` until the dependencies are merged:

- [x] https://github.com/google/ggrc-core/pull/8308

# Issue description

Add base_id column to ACL table that will be used in future acl read operation optimiziations

# Steps to test the changes

Check if the  base_id column matches the new ACL documentation and check that the app does not break.

# Solution description

Just added the base_id column but without a proper relationship for it because it should not be used with ORM.

# Sanity checklist



# Migration checklist


# PR Review checklist
